task_three_and_four/src/test/java/com/epam/webdriver/aleksandrgostev/driver/DriverManager.java

```diff
@@ -14,13 +16,26 @@ private DriverManager() {
     }
 
     public static WebDriver getDriver() {
-        if (driver.get() == null) {
-            driver.set(new ChromeDriver());
-
+        if (null == driver.get()) {
+//            //How to run test with Param=kkk via Intellij IDEA?
+//            System.out.println(System.getProperty("kkk"));
+            switch (System.getProperty("browser")) {
+                case "firefox": {
+                    WebDriverManager.firefoxdriver().setup();
+                    driver.set(new FirefoxDriver());
+                }
```

Comment:
_Optional comment_: just for your curiosity you may increase Java version to 14 in the main pom file and then try a new switch syntax